source/quick-start.txt

```diff
@@ -2,5 +2,175 @@
 Quick Start
 ===========
 
-
 .. default-domain:: mongodb
+
+Check for Node and NPM
+----------------------
+
+Ensure your system has Node greater than or equal to version 8 and
+NPM (Node Package Manager) installed. You can verify whether you have Node and
+NPM installed by executing the following in your terminal.
+
+First, check the version of Node
+
+.. code-block:: sh
+
+   node -v
+
+You should see output similar to the following:
+
+.. code-block:: sh
+
+   v12.14.1
+
+Next, check the version of NPM
+
+.. code-block:: sh
+
+   npm -v
+
+You should see output similar to the following:
+
+.. code-block:: sh
+
+   6.13.4
+
+If you do not see any output, you will need to install Node and NPM.
+Refer to `nodejs.org <https://nodejs.org>`_ for installation instructions.
+
+
+Initialize the Project
+----------------------
+
+The starting point for any Node application is initializing your project. Navigate
+to a new directory and initialize a new project.
+
+.. code-block:: sh
+
+  mkdir node_quickstart
+  cd node_quickstart
+  npm init -y
+
+.. admonition:: Why the -y?
+  :class: note
+
+  Specifying ``-y`` tells NPM to use default values. You can omit the ``-y``
+  flag to proceed through an interactive selection process for common project
+  settings.
+
+
+
+Add MongoDB as a Dependency
+---------------------------
+
+Next, add the Node MongoDB driver as a project dependency. Use the ``--save``
+flag to save the dependency to ``package.json`` file that was automatically
+generated in the previous step.
+
+.. code-block:: sh
+
+  npm install --save mongodb
+
+NPM will download dependencies and output information to your terminal. After
```

Comment:
Stick to present tense (this is future tense). Also, this is the first mention of "workspace" which can confuse a reader. Suggestion:
"NPM downloads the dependencies required for the installation and saves them to the ``node_modules`` directory in your project directory."